app/models/categorizer.ts

```diff
@@ -110,3 +110,11 @@ export enum CategorizerType {
   PaperTag = "PaperTag",
   PaperFolder = "PaperFolder",
 }
+
+export type CategorizeListItem = {
```

Comment:
I would suggest move these to `contextmenu-service.ts` as they are only used there.

Reply:
OK, then rename it with `CategorizerMenuItem`

Reply:
rename and put it in `categorizer.ts`, right?